airgun/widgets.py

```diff
@@ -71,7 +71,7 @@ class Search(Widget):
     default_result_locator = Text("//a[contains(., '%s')]")
 
     def fill(self, value):
-        self.search_field.fill(value)
+        return self.search_field.fill(value)
```

Comment:
`fill` is never expected to return something. Can you explain reason for that change?

Reply:
Actually `fill` is always expected to return a value. It should be `bool` whether something (at least one of fields) was filled or not. That's probably useful to determine whether there was a point in calling `fill` or all the fields had expected values already. Will we use that somewhere or not - hard to tell at this point, but at least let's follow default interface.